compiler/rustc_privacy/src/lib.rs

```diff
@@ -642,6 +639,21 @@ impl Visitor<'tcx> for EmbargoVisitor<'tcx> {
             }
             // Foreign modules inherit level from parents.
             hir::ItemKind::ForeignMod { .. } => self.prev_level,
+            hir::ItemKind::Macro(ref macro_def) => {
```

Comment:
```suggestion
            hir::ItemKind::Macro(MacroDef { macro_rules: true, .. }) => {
```
The non-`macro_rules` case is identical to other items below.